hack/.golint_failures

```diff
@@ -416,7 +416,6 @@ pkg/volume/git_repo
 pkg/volume/host_path
 pkg/volume/iscsi
 pkg/volume/local
-pkg/volume/nfs
```

Comment:
@linyouchong, makes it harder to review - the more packages you touch, the more approvers you need, so smaller PRs are better imo.

Reply:
There will be many PR if we do so.
According to https://github.com/kubernetes/kubernetes/issues/68026#issuecomment-418146719
We could put multiple commit in one PR.

Reply:
@linyouchong I have don several PRs for the volume package, and the reason I split it is exactly  because what @shubheksha mentioned. As one example for aws_ebs volume, I touched 20+ files and it requires 4+ approval to get it through, which is already tough. I would suggest to not combine all the volume fix in one PR.

Reply:
Since this PR is relatively small, I can try to cover some other pkgs like pkg/volume/local